mne/viz/backends/tests/test_abstract_notebook.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+#
+# Authors: Alex Rockhill <aprockhill@mailbox.org>
+
+# NOTE: Tests in this directory must be self-contained because they are
+# executed in a separate IPython kernel.
+
+import sys
+import pytest
+from mne.utils import check_version
+
+pytestmark = pytest.mark.skipif(
+    sys.platform.startswith('win') or not check_version('ipympl'),
+    reason='nbexec does not work on Windows')
+
+
+def test_widget_abstraction_notebook(nbexec):
+    """Test the GUI widgets abstraction in notebook."""
+    from mne.viz import set_3d_backend
+    from mne.viz.backends.renderer import _get_backend
```

Comment:
Why this instead of the public `mne.viz.get_3d_backend`?

Reply:
Confirmed, this just returns a string, you need the module. The name space is a bit confusing...